glean-core/python/glean/__init__.py

```diff
@@ -15,7 +15,7 @@
 __version__: str = "unknown"
 try:
     __version__ = str(get_distribution("glean-sdk").version)
-except DistributionNotFound:
+except DistributionNotFound:  # pragma: no cover
```

Comment:
Increase coverage by .... ignoring stuff :D

Reply:
That's the way to do it!